src/callbacks_step/analysis.jl

```diff
@@ -163,42 +163,42 @@ function initialize!(cb::DiscreteCallback{Condition, Affect!}, u_ode, du_ode, t,
 
         # write header of output file
         open(joinpath(output_directory, analysis_filename), "w") do io
-            @printf(io, "#%-8s", "timestep")
-            @printf(io, "  %-14s", "time")
-            @printf(io, "  %-14s", "dt")
+            print(io, "#timestep ")
```

Comment:
Shouldn't this be 
```suggestion
            print(io, "timestep ")
```
or
```suggestion
            print(io, "# timestep ")
```
? IIRC, the `#` was just a comment symbol, but now it looks like it's part of the field name

Reply:
I was not sure about this, I thought that the `#` should stand for "number", i.e, `#timesteps` = "number timesteps".

Reply:
Could you please check whether you can read the file using `readdlm` and describe the command in the NEWS.md? I would also have expected to use `# ` to indicate a comment/header

Reply:
Hm, so better bring this back?

Reply:
I think for flexibility of post-processing the analysis data it would be good to add back in the comment symbol.